providers/rackspace-cloudservers-us/src/test/java/org/jclouds/rackspace/cloudservers/us/compute/CloudServersUSTemplateBuilderLiveTest.java

```diff
@@ -39,11 +39,11 @@ public CloudServersUSTemplateBuilderLiveTest() {
 
    @Test
    public void testTemplateBuilder() {
-      Template defaultTemplate = this.view.getComputeService().templateBuilder().build();
+      Template defaultTemplate = this.view.getComputeService().templateBuilder().imageNameMatches(".*Ubuntu 14\\.10.*").build();
```

Comment:
I won't revert as some of the fixes (such as the OS version) I want to keep, but I will adjust it to make sure it works as intended. Thanks!